osversion/windowsbuilds.go

```diff
@@ -35,4 +35,7 @@ const (
 
 	// V20H2 corresponds to Windows Server 20H2 (semi-annual channel).
 	V20H2 = 19042
+
+	// V21H1 corresponds to Windows Server 21H1 (semi-annual channel).
+	V21H1 = 19043
```

Comment:
Some of the things here, like this, feel like they'd be better off in their own commit.

Reply:
I added a comment to the test to indicate why it blocks on this version

Reply:
ty